index.js

```diff
@@ -1,10 +1,13 @@
 'use strict';
 
 var mit = require('./configs/mit');
+var camunda = require('./configs/camunda');
 
 module.exports = {
   configs: {
     mit,
-    MIT: mit
+    MIT: mit,
+    camunda,
+    CAMUNDA: camunda
```

Comment:
I've provided the shortcut `MIT`, as `MIT` is what the license is called.

Is the Camunda licensed called `Camunda` or `CAMUNDA`? The shortcut should reflect that I think.

Reply:
The alternative is to not provide the `CAMUNDA` alias at all.